### DISCUSSION TITLE
Cannot restart nuxt:  Cannot read properties of undefined (reading 'resolvePath')

### DISCUSSION THREAD

**Discussion by alfredlua** [nuxt/content#1268] [Q&A]
Hi! I have an existing Nuxt project and I want to add Nuxt Content V1 to it. It seems like Nuxt Content V2 does not support static sites yet, so I thought I'd stick to V1.

I'm following the installation instructions on [the documentation](https://content.nuxtjs.org/v1/getting-started/installation). I installed using `npm install @nuxt/content@1.15.1`. 

But when I add the following code to my nuxt.config.ts file, I get an error:

```
{
  modules: [
    '@nuxt/content'
  ],
  content: {
    // Options
  }
}
```

```
 ERROR  Cannot restart nuxt:  Cannot read properties of undefined (reading 'resolvePath')                                                                                                                             15:10:46

  at Object.module.exports (node_modules/@nuxt/content/lib/index.js:57:54)
  at installModule (node_modules/@nuxt/kit/dist/index.mjs:398:20)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1336:7)
  at async load (node_modules/nuxt/node_modules/nuxi/dist/chunks/dev.mjs:6734:9)
  at async _applyPromised (node_modules/nuxt/node_modules/nuxi/dist/chunks/dev.mjs:6686:10)
```

Any help or suggestions will be appreciated. Thank you! 

Edit: Oh I'm on Nuxt 3. Could that be the issue? If that is the case, how could I use Nuxt Content V2 with my static site?

**Comment by ManasMadrecha** [comment]
For Nuxt 2, use Content V1. It has almost all the features of V2, but lacks composables, or different types of Content components, or MDC support (possibly available through [remark-mdc plugin](https://www.npmjs.com/package/@docus/remark-mdc-edge), customizable TOC scanning searchDepth, etc.).

For Nuxt 3, use Content V2. And wait for the static option to be implemented. 
EDIT: Check this https://github.com/danielroe/nuxt-full-static
